### PR TITLE
add katello-sam as an installable options

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

**Comment by komidore64** [comment]
this can't *really* be tested until we get the katello pipeline fixed because `katello-sam` and `sam-installer` aren't in the nightly repo yet

File: setup.rb [hunk 1]
```diff
@@ -207,8 +216,8 @@ def setup_koji_repos(os)
   system('rpm -e foreman-release')
   system('rpm -e katello-repos')
   system('rpm -e puppetlabs-release')
-  #system('cp ./rhscl-ruby193-el7-epel-7.repo /etc/yum.repos.d/')
-  #system('cp ./rhscl-v8314-el7-epel-7.repo /etc/yum.repos.d/')
+  # system('cp ./rhscl-ruby193-el7-epel-7.repo /etc/yum.repos.d/')
+  # system('cp ./rhscl-v8314-el7-epel-7.repo /etc/yum.repos.d/')
```
Comment (mccun934) [inline setup.rb]: whitespace changes?

File: setup.rb [hunk 2]
```diff
@@ -245,7 +254,7 @@ def setup_koji_repos(os)
   system('yum -y remove puppet')
 end
 
-#ensure puppet is installed
+# ensure puppet is installed
 system('yum -y update puppet')
```
Comment (mccun934) [inline setup.rb]: whitespace?

**Comment by mccun934** [comment]
ACK, installed fine in my test

**Comment by dustints** [comment]
saw this in my testing: 
/var/log/foreman/db_seed.log
```
rake aborted!
Please install the sqlite3 adapter: `gem install activerecord-sqlite3-adapter` (Could not find sqlite3 (~> 1.3.5) amongst [actionmailer-3.2.8, actionpack-3.2.8, activemodel-3.2.8, activerecord-3.2.8, activeresource-3.2.8, activesupport-3.2.8, algebrick-0.4.0, ancestry-2.0.0, anemone-0.7.2, angular-rails-templates-0.0.4, ansi-1.4.3, apipie-params-0.0.3, apipie-rails-0.2.6, archive-tar-minitar-0.5.2, arel-3.0.2, audited-3.0.0, audited-activerecord-3.0.0, bastion-0.2.0, bigdecimal-1.1.0, builder-3.0.0, bundler-1.3.5, bundler_ext-0.3.0, coffee-rails-3.2.2, coffee-script-2.2.0, coffee-script-source-1.3.3, commonjs-0.2.7, daemons-1.1.4, deep_cloneable-2.0.2, deface-0.7.2, diff-lcs-1.1.3, docker-api-1.17.0, dynflow-0.7.6, erubis-2.7.0, excon-0.43.0, execjs-1.4.0, fast_gettext-0.9.2, fog-1.25.0, fog-core-1.25.0, fog-json-1.0.0, fog-xml-0.1.1, foreigner-1.7.1, foreman-tasks-0.6.12, foreman_docker-1.0.0, foreman_sam-0.0.1, formatador-0.2.1, friendly_id-4.0.10.1, gettext_i18n_rails-1.2.0, haml-4.0.6, haml-rails-0.4, hashr-0.0.22, hike-1.2.1, hooks-0.2.2, hpricot-0.8.6, i18n-0.6.0, i18n_data-0.2.7, io-console-0.3, ipaddress-0.8.0, journey-1.0.4, jquery-ui-rails-4.1.2, json-1.5.5, justified-0.0.4, katello-2.2.0, ldap_fluff-0.3.3, less-2.5.1, less-rails-2.5.0, little-plugger-1.1.3, logging-1.8.1, mail-2.4.4, mime-types-1.19, minitest-2.5.1, multi_json-1.10.1, net-http-persistent-2.7, net-ldap-0.10.0, net-scp-1.1.0, net-ssh-2.6.7, netrc-0.7.7, nokogiri-1.5.11, oauth-0.4.7, polyglot-0.3.3, qpid_messaging-0.26.1, rabl-0.11.4, rack-1.4.1, rack-cache-1.2, rack-jsonp-1.3.1, rack-protection-1.2.0, rack-ssl-1.3.2, rack-test-0.6.1, rails-3.2.8, railties-3.2.8, rake-0.9.2.2, rdoc-3.9.5, rest-client-1.6.7, robotex-1.0.0, ruby2ruby-2.1.3, ruby_parser-3.6.3, runcible-1.3.1, safemode-1.2.2, sass-3.4.9, scoped_search-2.7.1, secure_headers-1.4.1, sequel-4.17.0, sexp_processor-4.4.4, sinatra-1.3.2, sprockets-2.10.1, sprockets-rails-2.0.1, strong_parameters-0.2.1, thor-0.18.1, tilt-1.3.3, tire-0.6.2, treetop-1.4.10, turbolinks-2.5.3, tzinfo-0.3.33, unf-0.1.3, unf_ext-0.0.6, validates_lengths_from_database-0.4.0, wicked-1.1.0, will_paginate-3.0.2])

Tasks: TOP => db:migrate => environment
(See full trace by running task with --trace)
rake aborted!
```
going to investigate some more.

**Comment by ehelms** [comment]
That error exists on all installs. Don't bother chasing that rabbit.

File: setup.rb (hunk 1 above)
Comment (komidore64) [inline setup.rb]: yeah, i can remove if needed

**Comment by dustints** [comment]
@ehelms ah ok,  ACK from me too!

**Pull request merged**